packages/create-atomic/plopfile.ts

```diff
@@ -0,0 +1,62 @@
+import {NodePlopAPI} from 'plop';
+import {spawn} from 'child_process';
+import {getPackageManager} from './utils';
+
+interface PromptsAnswers {
+  project: string;
+}
+
+module.exports = function (plop: NodePlopAPI) {
```

Comment:
IDK if plop support ESM exports. If yes:
```suggestion
export default function (plop: NodePlopAPI) {
```

If it doesn't support it (or if if just ain't working), do this:
```suggestion
export = function (plop: NodePlopAPI) {
```
See: https://github.com/Microsoft/TypeScript-Handbook/blob/master/pages/Modules.md#export--and-import--require